namesrv/src/main/java/org/apache/rocketmq/namesrv/routeinfo/RouteInfoManager.java

```diff
@@ -527,83 +529,81 @@ public void unregisterBroker(
 
     public void unRegisterBroker(Set<UnRegisterBrokerRequestHeader> unRegisterRequests) {
         try {
-            try {
-                Set<String> removedBroker = new HashSet<>();
-                Set<String> reducedBroker = new HashSet<>();
-                Map<String, BrokerStatusChangeInfo> needNotifyBrokerMap = new HashMap<>();
+            Set<String> removedBroker = new HashSet<>();
+            Set<String> reducedBroker = new HashSet<>();
+            Map<String, BrokerStatusChangeInfo> needNotifyBrokerMap = new HashMap<>();
```

Comment:
code formate, you can read this document https://rocketmq.apache.org/docs/%E8%B4%A1%E7%8C%AE%E6%8C%87%E5%8D%97/30code-guidelines